src/dapp-tests/fail/invariantFail.sol

```diff
@@ -24,3 +24,55 @@ contract TestdappTest is DSTest {
         assertLt(testdapp.x(), 100);
     }
 }
+
+contract BrokenCoin {
+  mapping(address=>uint) public balanceOf;
+  constructor(uint amount) public {
+    balanceOf[msg.sender] = amount;
+  }
+
+  function transfer(address to, uint amount) public {
+    uint senderBal = balanceOf[msg.sender];
+    uint toBal = balanceOf[to];
+    require(senderBal >= amount);
+    senderBal += amount;
+    toBal -= amount;
+    balanceOf[msg.sender] = senderBal;
+    balanceOf[to] = toBal;
+  }
+}
+
+contract InvariantTest is DSTest {
+    BrokenCoin token;
+    User user;
+    address[] targetContracts_;
```

Comment:
why not `address[] public targetContracts`?

Reply:
because that makes a function `targetContracts(uint)(address)`

Reply:
did not know that trick!